frontend/src/components/not-found/not-found.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import styles from './not-found.module.scss';
+import iconLogo from 'assets/images/icon-logo.svg';
+import clsx from 'clsx';
+
+const NotFound: React.FC = () => {
+  const currentYear = new Date().getFullYear();
+
+  return (
+    <>
+      <div className={clsx(styles.notFoundContainer, styles.flexColumnCentered)}>
+        <h1 className={styles.textNotFound}><span className={styles.text404}>404</span> Page not found</h1>
+      </div>
+      <footer className={clsx(styles.footerContainer, styles.flexColumnCentered)}>
+        <img src={iconLogo} height="31.76px" width="35px" loading="lazy" alt="HealthCareLogo" />
```

Comment:
```suggestion
        <img src={iconLogo} height="32" width="35" loading="lazy" alt="HealthCareLogo" />
```
px and points aren't allowed in the attrs